setup.py

```diff
@@ -12,15 +12,29 @@
 import platform
 import re
 import ast
+import sys
 from setuptools import find_packages, setup
 from setuptools.extension import Extension
 from setuptools.command.build_ext import build_ext as _build_ext
 
+# numpy is not required with help or information display options
+if any(
+    arg.startswith('--') and arg not in [
+        '--verbose',
+        '--quiet',
+        '--dry-run',
+        '--no-user-cfg',
+        '--command-packages'] for arg in sys.argv):
+    setup_requires = []
+else:
+    setup_requires = ['numpy >= 1.9.2']
```

Comment:
You'll need to modify the `setup()` call below to use this `setup_requires` variable.